cmd/agent/install_mac_os.sh

```diff
@@ -438,6 +448,15 @@ else
     $sudo_cmd launchctl kickstart "system/$service_name"
 fi
 
+# Cleanup tmp files used for installation
+if [ -f /tmp/install-ddagent/system-wide ]; then
+  rm /tmp/install-ddagent/system-wide
```

Comment:
wouldn't this make more sense in the `postinst` script, once it's been read? I don't really know if there's a strict ordering between the 2 scripts though

Reply:
If it's put in the `postinst` script older versions of the agent won't cleanup these files right ?